src/components/AddItem.jsx

```diff
@@ -0,0 +1,184 @@
+			setIsNotAdded(true);
+			setItemName('');
+			setDays(0);
+			return;
+		}
+
+		if (checkForDuplicates) {
+			setIsAlreadyAdded(true);
+			setItemName('');
+			setDays(0);
+			return;
+		}
+
+		let itemData = {
+			itemName: itemName,
+			daysUntilNextPurchase: days,
+		};
+
+		try {
+			await addItem(listToken, itemData);
+			setIsAdded(true);
+			setItemName('');
+			setDays(0);
+		} catch (error) {
+			console.log(error);
+		}
+	};
+
+	return (
+		<div>
+			<Button
+				leftIcon={<AddIcon />}
+				bg="brand.off_white"
+				color="brand.navy"
+				onClick={() => {
+					setIsAdded(false);
+					setIsNotAdded(false);
+					setIsAlreadyAdded(false);
+					onOpen();
+				}}
+			>
+				Add Item
+			</Button>
+
+			{/* Alert that displays when item is added */}
+			{isAdded && (
+				<Alert status="success">
+					<AlertIcon />
+					Item successfully added to your list!
+				</Alert>
+			)}
+
+			{/* Alert that displays when item not added because missing field/s */}
+			{isNotAdded && (
+				<Alert status="error">
+					<AlertIcon />
+					Item was NOT added because one of more fields were incomplete. Please
+					fill out all fields.
+				</Alert>
+			)}
+
+			{/* Alert that displays when item not added because duplicate */}
+			{isAlreadyAdded && (
+				<Alert status="warning">
+					<AlertIcon />
+					You've already added this item, so we were not able to add it to your
+					list.
+				</Alert>
+			)}
+
+			<Modal isOpen={isOpen} onClose={onClose}>
+				<ModalOverlay />
+				<ModalContent bg="brand.off_white" color="brand.navy">
+					<ModalHeader>Add an item to your list</ModalHeader>
+					<ModalCloseButton />
+					<ModalBody>
+						<form id="addItemForm" onSubmit={handleSubmit}>
+							<FormControl isInvalid={isEmptyName}>
+								<FormLabel fontWeight="bold">Item name:</FormLabel>
+								<Input
+									type="text"
+									id="itemName"
+									onChange={handleItemNameChange}
+									value={itemName}
+								/>
+								{isEmptyName && (
+									<FormErrorMessage>Item name is required.</FormErrorMessage>
+								)}
+							</FormControl>
+							<br />
+							<FormControl isInvalid={isEmptyDays}>
+								<FormLabel fontWeight="bold">
+									How soon will you buy this again?
+								</FormLabel>
+								<Select
+									placeholder="Choose one"
+									onChange={handleDaysChange}
+									value={days}
+								>
+									<option value="7">Soon</option>
+									<option value="14">Kind of Soon</option>
+									<option value="30">Not Soon</option>
+								</Select>
+								{isEmptyDays && (
+									<FormErrorMessage>A selection is required.</FormErrorMessage>
+								)}
+							</FormControl>
+						</form>
+					</ModalBody>
+
+					<ModalFooter>
+						<Button
+							leftIcon={<AddIcon />}
+							form="addItemForm"
+							type="submit"
+							onClick={onClose}
+							bg="brand.navy"
+							color="brand.off_white"
+						>
+							Add Item
+						</Button>
+					</ModalFooter>
```

Comment:
I recommend considering a slight modification to the button's visibility. Currently, when hovered upon, it turns white, making it difficult to discern the text "Add Item."

As you mentioned in your note, it's essential for all of us to reach a consensus on the button's size and styling.

<img width="535" alt="Screenshot 2023-10-04 at 6 47 13 PM" src="https://github.com/the-collab-lab/tcl-62-smart-shopping-list/assets/49603115/783f15e4-b790-4142-bebc-c516590abe2b">